### ISSUE TITLE
Pretrained model?

### ISSUE THREAD

**Issue by davidmartinrius** [Anvarjon/Age-Gender-Classification#1]
Hi! How are you doing?

I was wondering if you were planning to share a pretraibed model, a SOTA?

Thank you!

**Comment by GGbboy** [comment]
this is

**Comment by Anvarjon** [comment]
Hello! 

I can build, train, and share pre-trained models here, but I am not sure I will get the exact results shown in the paper. Because I lost the models that gave the results shown in the paper. I will share pre-trained models this weekend. 

Thank you!

**Comment by Anvarjon** [comment]
Hello,

I have shared the link to pre-trained models for age, gender, and gender_age classification tasks. Additionally, the inference code is also uploaded. Please let me know if you have any issues using the shared models. For now, I will close this issue.

Thank you!

**Comment by davidmartinrius** [comment]
Great Mark! What dataset did you use to train the model? This way I can test the inference accuracy with seen and unseen samples. Thanks!

**Comment by davidmartinrius** [comment]
Ah I read it in the README.md , common voice.
